src/coreComponents/constitutive/fluid/multifluid/CO2Brine/PhaseModel.hpp

```diff
@@ -58,22 +58,26 @@ struct PhaseModel
               array1d< array1d< string > > const & inputParams,
               string_array const & componentNames,
               array1d< real64 > const & componentMolarWeight,
-              bool const printTable )
+              bool const printInCsv,
+              bool const printInLog )
```

Comment:
Add the missing comments:
```suggestion
   * @param[in] printInCsv ...
   * @param[in] printInLog ...
```